tutorial.ipynb

```diff
@@ -341,6 +354,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "One thing to note here is that unlike the earlier plots, these plots are not smooth curves, this is because the load is piecewise, not constant. You see jumps in the plots at the times when the load changes. Also, the simulation is long enough for the temperature to reach an equilibrium. \n",
+    "\n",
```

Comment:
I think these descriptions are of great value. They also appear to have just the right amount of information (i.e. not too much but not too little)